test/src/test/java/hudson/model/AbstractItemTest.java

```diff
@@ -40,4 +59,87 @@ public void reload() throws Exception {
         assertEquals(b.getDescription(), b2.getDescription()); // but should have the same properties
     }
 
+    @Test
+    public void rename() throws Exception {
```

Comment:
None of these test for just a pure rename?

Reply:
You're right, I didn't update the name of the test after I changed them to just calling `doCheckNewName`. I'll fix them now, thanks!